PubSub/src/Topic.php

```diff
@@ -763,7 +838,30 @@ private function formatMessage($message)
             );
         }
 
-        return $message;
+        return $this->serializer->decodeMessage(
+            new PubsubMessage(),
+            $message
+        );
+    }
+
+    private function formatSchemaSettings(array &$data)
+    {
+        if (isset($data['schemaSettings'])) {
+            $enc = $data['schemaSettings']['encoding'] ?? Encoding::ENCODING_UNSPECIFIED;
```

Comment:
for reference, this is an example of the null-coalescing operator (e.g. `??`) being used correctly. If we used `?:` here, we could potentially receive an "undefined array key" error.